### ISSUE TITLE
[Feature request] Reorganize execution steps in workflows

### ISSUE THREAD

**Issue by K3UL** [oliverschwendener/ueli#321]
Hi,

I think it would be nice to be able to reorganize steps (drag and drop for example) in the execution steps of the workflows. Or at least it would be good to be able to insert a new step at a particular place, between two others.

Currently, to add a new step before all existing ones, if I'm not wrong, you have to remove all the steps and do them all over again.

**Comment by oliverschwendener** [comment]
Yes you are right, at the moment there is no option to reorganize the steps. Is there a reason why you want to do that? All steps are executed simultaneously and the order is not really relevant.

**Comment by K3UL** [comment]
Yes I actually thought they were executed in a sequence, my bad. I think the notion of "Execution step" is a little misleading in that matter.

I guess my feature request would then be :
- Make it possible to have the steps order matter (with a switch for a given workflow)
- Make it possible to reorganize steps for that case
- Make it possible to add delays between steps

I think this (control order and delay) can apply to a lot of use cases and can add some real power to the worflow plugin.

Beside that, I just want to say I have checked out several launchers and I really like what you've done with yours. I was trying to see if there was something like Alfred but for Windows and found yours infinitely more intuitive than Alfred.
